packages/node/src/dependencies.ts

```diff
@@ -102,7 +102,7 @@ export const buildRedirectHandler = (
 /**
  *
  * @param dependencies
- * @deprecated This function will be removed from the external API in an upcoming release.
+ * @hidden
```

Comment:
I would suggest keeping the `@deprecated` jsdoc as well as hidden will only hide it from our documentation - not the exports iirc.

Reply:
Correct me if I'm wrong, but given this is no longer exported from either `index.ts`, nor is `dependencies.ts` mapped in the `package.json` exports, then in order to import this you'd need to import it directly from `@inrupt/solid-client-authn-node/src/dependencies`, which you can technically do but isn't public API, right?